blivet/formats/lvmpv.py

```diff
@@ -107,6 +110,17 @@ def __repr__(self):
                "pe_start": self.pe_start, "data_alignment": self.data_alignment})
         return s
 
+    @property
+    def grow_to_fill(self):
```

Comment:
Please add a docstring here explaining how this works, e.g. what target size needs to be set to make this work and that `ActionResizeFormat` still needs to be created and executed.

Reply:
Fixed. Added docstring.